src/interfaces/index.ts

```diff
@@ -0,0 +1,225 @@
+export interface IPost {
```

Comment:
I suggest you to follow one interface naming style (either with or without `I`-prefix)

Reply:
Have fixed/ please check again